com.unity.render-pipelines.universal/Shaders/SpatialMappingOcclusion.shader

```diff
@@ -0,0 +1,78 @@
+Shader "Universal Render Pipeline/SpatialMappingOcclusion"
+{
+    Properties
+    {
+    }
+    SubShader
+    {
+        Tags {"RenderType" = "Opaque" "IgnoreProjector" = "False" "RenderPipeline" = "UniversalPipeline" "ShaderModel"="2.0"}
+        LOD 100
+
+        Blend One OneMinusSrcAlpha
+        ZWrite On
+        ZTest LEqual
+        Colormask 0
+        Cull Off
+
+        Pass
+        {
+            Name "Spatial Mapping Occlusion"
+
+            HLSLPROGRAM
+            #pragma exclude_renderers gles gles3 glcore
+            #pragma target 4.5
```

Comment:
same as other spatial shader, these requirements seem very high, make sure these line up with the original shader not the Lit forward pass